### PR TITLE
Create middle section and contact page

### PR DESCRIPTION
**Pull request by ElsonOtake**
Created four pages of projects, in these pages, the CSS grid was as a container and the Flexbox was one of the CSS grid items.
Created a page for contacts with the input text box and arranged using Flexbox.
Template number four of Figma was used as the model for this project.

### CODE REVIEW

File: project01.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <li class="proj_lang themefff"><a href="#">Css</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">html</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Javascript</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Ruby</a></li>
+    </ul>
+  </div>
+  <button type="button" class="proj_button theme44">See project</button>
```
Comment (Lucas-Masaba) [inline project01.html:33]: - [ ] As a core requirement, everything here should be in a `<section>` tag. Please surround all the content in a `<section>` tag 👍

File: project02.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <li class="proj_lang themefff"><a href="#">Css</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">html</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Javascript</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Ruby</a></li>
+    </ul>
+  </div>
+  <button type="button" class="proj_button theme44">See project</button>
```
Comment (Lucas-Masaba) [inline project02.html:31]: - [ ] As a core requirement, everything here should be in a `<section>` tag. Please surround all the content with a `<section>` tag 😎

File: project04.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <li class="proj_lang themefff"><a href="#">Css</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">html</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Javascript</a></li>
+      <li class="vl"></li>
+      <li class="proj_lang themefff"><a href="#">Ruby</a></li>
+    </ul>
+  </div>
+  <button type="button" class="proj_button theme44">See project</button>
```
Comment (Lucas-Masaba) [inline project04.html:31]: - [ ] As a core requirement, everything here should be in a `<section>` tag. Please surround all the content in a `<section>` tag 🙏

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @ElsonOtake  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Your README looks professional 👍 
- All linter checks are passing ✔️ 
- Design is following the of the Figma template 🥳 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- I realised you split all the projects and the contact sections into separate HTML files 😱 . I would like to let you know that this is incorrect ❌. You are expected to place the projects and contact sections preferably in the `index.html` file. You can do this much later after this project has been approved 👍

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @ElsonOtake ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# Required Changes ♻️

- [ ] Please use one file for all HTML structure. :+1: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ]  In the readme file, please add a live demo link to your project. :+1: 

- [ ] Your page is not matched with the Figma design. Please try to match your design with the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
f1 | y1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by aimemalaika — APPROVED** [review]
Hi @ElsonOtake , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**